examples/connection_pools/Rocket.toml

```diff
@@ -0,0 +1,9 @@
+[[global.databases]]
+name = "my_sqlite_db"
+adapter = "diesel_sqlite"
+url = "db.sqlite"
```

Comment:
We should also make the pool size configurable with a sane default.

Reply:
Perhaps the default should be a multiple of the number of workers. While we're synchronous, the most we can ever use is exactly `workers`, so there's no reason to go beyond this. Once we're aync, I think we're going to have to rethink things just a bit here. We'll get to that when we get to that.

Reply:
Good idea. Makes sense that more than the number of workers doesn't make sense given the synchronous nature of the framework right now.